package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "speaker",
-  "version": "0.4.0",
+  "version": "0.4.1",
```

Comment:
Please don't bump the version in the PR, that's better left to the release process

Reply:
Thanks for the replies.

For the version bump, I've worked on other projects where they did that.  I'll take it out.

I was thinking that the call-back would loose context (I haven't worked with libuv much), but I can remove "THIS" if it's not needed.

For the naming conventions, there are a lot of styles out there.  Is there an example style that I should use?  WRT the actual data fields included in the event, I just put in there what I needed for my little use case, but other data can be added if appropriate.  Or should it not put include any data, and then the receiver can just pull the data out of the object?